packages/devtools/lib/src/ui/primer.dart

```diff
@@ -63,11 +63,15 @@ class PButton extends CoreElement {
     setAttribute('type', 'button');
   }
 
-  PButton.icon(String text, Icon icon, {String title, List<String> classes})
+  PButton.icon(String text, Icon icon,
+      {String title, List<String> classes, bool invertDark = false})
       : super('button', classes: 'btn optional-text') {
     setAttribute('type', 'button');
     setAttribute('title', title ?? text);
     classes?.forEach(clazz);
+    if (invertDark) {
```

Comment:
put this logic directly in the Icon class instead of when creating the button because these icons always need to display differently in a dark theme because conceptually they should match the foreground color in both the light and dark theme because they are black and white icons. 
invertDark is an implementation detail. The core concept is we are trying to match the foreground color so I would expose that in the api and in the implementation in the html_icons.dart file expose the implementation achieved by adding CSS styles to invert and later colorize.

Reply:
Done!